Automattic/woocommerce-payments
# Add events for interactions with transactions page

Fixes #7961

#### Changes proposed in this Pull Request
Track the view of the transaction page and when the user press the filter button on the transaction page.
We use
- `wcadmin_page_view` event with the path `payments_transactions`( `path: payments_transactions` ) for transactions listing
- for clicking the filter button we add `filter: advanced` property:
```
filter: advanced
path: payments_transactions
```









#### Testing instructions
- Run `localStorage.setItem( 'debug', 'wc-admin:tracks' );` on your site browser console. And look for `verbose` logs.
- Visit the WC Pay transactions page and confirm that the `wcadmin_page_view` event fires when entering the ransaction page
- Add an advanced press the filter button. Confirm that the `wcadmin_page_view` with `filter: advanced param` event fires




*

-------------------


**Post merge**
